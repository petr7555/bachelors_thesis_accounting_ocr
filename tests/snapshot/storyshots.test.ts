import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  // from project root
  configPath: './.storybook-rn',
  // optional
  framework: 'react-native',
});
