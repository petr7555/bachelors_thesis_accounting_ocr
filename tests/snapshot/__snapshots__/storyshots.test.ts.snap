// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots ScanButton default 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  >
    <View
      pointerEvents="box-none"
      style={
        Object {
          "zIndex": 1,
        }
      }
    >
      <View
        accessibilityElementsHidden={false}
        importantForAccessibility="auto"
        onLayout={[Function]}
        pointerEvents="box-none"
        style={null}
      >
        <View
          pointerEvents="box-none"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "zIndex": 0,
            }
          }
        >
          <View
            style={
              Object {
                "backgroundColor": "rgb(255, 255, 255)",
                "borderBottomColor": "rgb(216, 216, 216)",
                "flex": 1,
                "shadowColor": "rgb(216, 216, 216)",
                "shadowOffset": Object {
                  "height": 0.5,
                  "width": 0,
                },
                "shadowOpacity": 0.85,
                "shadowRadius": 0,
              }
            }
          />
        </View>
        <View
          pointerEvents="box-none"
          style={
            Object {
              "height": 64,
              "maxHeight": undefined,
              "minHeight": undefined,
              "opacity": undefined,
              "transform": undefined,
            }
          }
        >
          <View
            pointerEvents="none"
            style={
              Object {
                "height": 20,
              }
            }
          />
          <View
            pointerEvents="box-none"
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "center",
              }
            }
          >
            <View
              pointerEvents="box-none"
              style={
                Object {
                  "marginHorizontal": 16,
                  "opacity": 1,
                }
              }
            >
              <Text
                accessibilityRole="header"
                aria-level="1"
                numberOfLines={1}
                onLayout={[Function]}
                style={
                  Object {
                    "color": "rgb(28, 28, 30)",
                    "fontSize": 17,
                    "fontWeight": "600",
                  }
                }
              >
                MockedScreen
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      onLayout={[Function]}
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        pointerEvents="box-none"
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          collapsable={false}
          style={
            Object {
              "opacity": 1,
            }
          }
        />
        <View
          accessibilityElementsHidden={false}
          closing={false}
          gestureVelocityImpact={0.3}
          importantForAccessibility="auto"
          onClose={[Function]}
          onGestureBegin={[Function]}
          onGestureCanceled={[Function]}
          onGestureEnd={[Function]}
          onOpen={[Function]}
          onTransition={[Function]}
          pointerEvents="box-none"
          style={
            Array [
              Object {
                "overflow": undefined,
              },
              Object {
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              },
            ]
          }
          transitionSpec={
            Object {
              "close": Object {
                "animation": "spring",
                "config": Object {
                  "damping": 500,
                  "mass": 3,
                  "overshootClamping": true,
                  "restDisplacementThreshold": 10,
                  "restSpeedThreshold": 10,
                  "stiffness": 1000,
                },
              },
              "open": Object {
                "animation": "spring",
                "config": Object {
                  "damping": 500,
                  "mass": 3,
                  "overshootClamping": true,
                  "restDisplacementThreshold": 10,
                  "restSpeedThreshold": 10,
                  "stiffness": 1000,
                },
              },
            }
          }
        >
          <View
            needsOffscreenAlphaCompositing={false}
            pointerEvents="box-none"
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <View
              collapsable={false}
              forwardedRef={[Function]}
              onGestureHandlerEvent={[Function]}
              onGestureHandlerStateChange={[Function]}
              style={
                Object {
                  "flex": 1,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateX": 0,
                    },
                  ],
                }
              }
            >
              <View
                pointerEvents="none"
                style={
                  Object {
                    "backgroundColor": "rgb(242, 242, 242)",
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "shadowColor": "#000",
                    "shadowOffset": Object {
                      "height": 1,
                      "width": -1,
                    },
                    "shadowOpacity": 0.3,
                    "shadowRadius": 5,
                    "top": 0,
                    "width": 3,
                  }
                }
              />
              <View
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "overflow": "hidden",
                    },
                    Array [
                      Object {
                        "backgroundColor": "rgb(242, 242, 242)",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                      "flexDirection": "column-reverse",
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "flex": 1,
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "justifyContent": "center",
                        }
                      }
                    >
                      <View
                        accessible={true}
                        focusable={true}
                        onClick={[Function]}
                        onResponderGrant={[Function]}
                        onResponderMove={[Function]}
                        onResponderRelease={[Function]}
                        onResponderTerminate={[Function]}
                        onResponderTerminationRequest={[Function]}
                        onStartShouldSetResponder={[Function]}
                        style={
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "#310783",
                            "borderColor": "#078331",
                            "borderRadius": 100,
                            "borderWidth": 5,
                            "height": 90,
                            "justifyContent": "center",
                            "marginBottom": 10,
                            "opacity": 1,
                            "width": 90,
                          }
                        }
                      >
                        <Text
                          allowFontScaling={false}
                          style={
                            Array [
                              Object {
                                "color": undefined,
                                "fontSize": 12,
                              },
                              Object {
                                "color": "white",
                                "fontSize": 50,
                              },
                              Object {
                                "fontFamily": "Ionicons",
                                "fontStyle": "normal",
                                "fontWeight": "normal",
                              },
                              Object {},
                            ]
                          }
                        >
                          
                        </Text>
                      </View>
                      <Modal
                        animationType="none"
                        deviceHeight={null}
                        deviceWidth={null}
                        hardwareAccelerated={false}
                        hideModalContentWhileAnimating={false}
                        onBackdropPress={[Function]}
                        onModalHide={[Function]}
                        onModalWillHide={[Function]}
                        onModalWillShow={[Function]}
                        onRequestClose={[Function]}
                        scrollHorizontal={false}
                        scrollOffset={0}
                        scrollOffsetMax={0}
                        scrollTo={null}
                        supportedOrientations={
                          Array [
                            "portrait",
                            "landscape",
                          ]
                        }
                        swipeThreshold={100}
                        transparent={true}
                        visible={false}
                      >
                        <View
                          accessible={true}
                          focusable={true}
                          forwardedRef={[Function]}
                          onClick={[Function]}
                          onResponderGrant={[Function]}
                          onResponderMove={[Function]}
                          onResponderRelease={[Function]}
                          onResponderTerminate={[Function]}
                          onResponderTerminationRequest={[Function]}
                          onStartShouldSetResponder={[Function]}
                          style={
                            Object {
                              "backgroundColor": "black",
                              "bottom": 0,
                              "height": 1334,
                              "left": 0,
                              "opacity": 0,
                              "position": "absolute",
                              "right": 0,
                              "top": 0,
                              "width": 750,
                            }
                          }
                        />
                        <View
                          deviceHeight={null}
                          deviceWidth={null}
                          forwardedRef={[Function]}
                          hideModalContentWhileAnimating={false}
                          onBackdropPress={[Function]}
                          onModalHide={[Function]}
                          onModalWillHide={[Function]}
                          onModalWillShow={[Function]}
                          pointerEvents="box-none"
                          scrollHorizontal={false}
                          scrollOffset={0}
                          scrollOffsetMax={0}
                          scrollTo={null}
                          style={
                            Object {
                              "flex": 1,
                              "justifyContent": "flex-end",
                              "margin": 0,
                              "marginBottom": 100,
                              "transform": Array [
                                Object {
                                  "translateY": 0,
                                },
                              ],
                            }
                          }
                          supportedOrientations={
                            Array [
                              "portrait",
                              "landscape",
                            ]
                          }
                          swipeThreshold={100}
                        >
                          <View
                            style={
                              Object {
                                "alignItems": "center",
                                "backgroundColor": "white",
                                "borderColor": "#078331",
                                "borderRadius": 4,
                                "justifyContent": "center",
                                "padding": 22,
                              }
                            }
                          >
                            <View
                              style={
                                Array [
                                  Object {
                                    "borderRadius": 3,
                                    "overflow": "hidden",
                                  },
                                  Object {
                                    "borderRadius": 3,
                                  },
                                  Object {
                                    "marginVertical": 5,
                                  },
                                  false,
                                ]
                              }
                            >
                              <View
                                accessibilityRole="button"
                                accessibilityState={
                                  Object {
                                    "busy": false,
                                    "disabled": false,
                                  }
                                }
                                accessible={true}
                                focusable={true}
                                onClick={[Function]}
                                onResponderGrant={[Function]}
                                onResponderMove={[Function]}
                                onResponderRelease={[Function]}
                                onResponderTerminate={[Function]}
                                onResponderTerminationRequest={[Function]}
                                onStartShouldSetResponder={[Function]}
                                style={
                                  Object {
                                    "opacity": 1,
                                  }
                                }
                              >
                                <View
                                  style={
                                    Object {
                                      "alignItems": "center",
                                      "backgroundColor": "#310783",
                                      "borderColor": "#2089dc",
                                      "borderRadius": 25,
                                      "borderWidth": 0,
                                      "flexDirection": "row",
                                      "height": 50,
                                      "justifyContent": "center",
                                      "padding": 8,
                                      "width": 690,
                                    }
                                  }
                                >
                                  <Text
                                    allowFontScaling={false}
                                    style={
                                      Array [
                                        Object {
                                          "color": undefined,
                                          "fontSize": 12,
                                        },
                                        Object {
                                          "color": "white",
                                          "fontSize": 28,
                                          "left": 40,
                                          "position": "absolute",
                                        },
                                        Object {
                                          "fontFamily": "Ionicons",
                                          "fontStyle": "normal",
                                          "fontWeight": "normal",
                                        },
                                        Object {},
                                      ]
                                    }
                                  >
                                    
                                  </Text>
                                  <Text
                                    style={
                                      Object {
                                        "color": "white",
                                        "fontSize": 18,
                                        "paddingVertical": 1,
                                        "textAlign": "center",
                                      }
                                    }
                                  >
                                    Take an image
                                  </Text>
                                </View>
                              </View>
                            </View>
                            <View
                              style={
                                Array [
                                  Object {
                                    "borderRadius": 3,
                                    "overflow": "hidden",
                                  },
                                  Object {
                                    "borderRadius": 3,
                                  },
                                  Object {
                                    "marginVertical": 5,
                                  },
                                  false,
                                ]
                              }
                            >
                              <View
                                accessibilityRole="button"
                                accessibilityState={
                                  Object {
                                    "busy": false,
                                    "disabled": false,
                                  }
                                }
                                accessible={true}
                                focusable={true}
                                onClick={[Function]}
                                onResponderGrant={[Function]}
                                onResponderMove={[Function]}
                                onResponderRelease={[Function]}
                                onResponderTerminate={[Function]}
                                onResponderTerminationRequest={[Function]}
                                onStartShouldSetResponder={[Function]}
                                style={
                                  Object {
                                    "opacity": 1,
                                  }
                                }
                              >
                                <View
                                  style={
                                    Object {
                                      "alignItems": "center",
                                      "backgroundColor": "#310783",
                                      "borderColor": "#2089dc",
                                      "borderRadius": 25,
                                      "borderWidth": 0,
                                      "flexDirection": "row",
                                      "height": 50,
                                      "justifyContent": "center",
                                      "padding": 8,
                                      "width": 690,
                                    }
                                  }
                                >
                                  <Text
                                    allowFontScaling={false}
                                    style={
                                      Array [
                                        Object {
                                          "color": undefined,
                                          "fontSize": 12,
                                        },
                                        Object {
                                          "color": "white",
                                          "fontSize": 28,
                                          "left": 40,
                                          "position": "absolute",
                                        },
                                        Object {
                                          "fontFamily": "Ionicons",
                                          "fontStyle": "normal",
                                          "fontWeight": "normal",
                                        },
                                        Object {},
                                      ]
                                    }
                                  >
                                    
                                  </Text>
                                  <Text
                                    style={
                                      Object {
                                        "color": "white",
                                        "fontSize": 18,
                                        "paddingVertical": 1,
                                        "textAlign": "center",
                                      }
                                    }
                                  >
                                    Select from gallery
                                  </Text>
                                </View>
                              </View>
                            </View>
                          </View>
                        </View>
                      </Modal>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RNCSafeAreaProvider>
</View>
`;
